### ISSUE TITLE
Break up reports by role

### ISSUE THREAD

**Issue by mgifford** [CivicActions/purple-hats#1]
"Break them up into separate reports for things that are the responsibility of the dev team, and for accessibility issues that are in user-created content."

We should be able to focus in on what content is primarily related to content authors vs produced via the CMS.  This is a basic level of awareness would allow teams to triage problems. 

It is safe to assume that everything within `<main>` or with a `role="main"` is controlled by the content author most of the time. 

We might be able to pick out certain content types to pick out landing pages, forms or views which are outside the direct control of authors. May be worth tagging the HTML so that it is easier to identify.

**Comment by mgifford** [comment]
WCAG issues are now broken down by role so that this is visible in the report.

I would like to still be able to filter based on role to highlight this further.
